src/tests/api.test.ts

```diff
@@ -0,0 +1,424 @@
+    const lang = 'hr';
+    const { data } = await ss.nodes(nodes).search({ lang });
+
+    expect(data[0].languageId).toEqual(lang);
+  });
+
+  it('should fetch a specified node', async () => {
+    const { data } = await ss.nodes(nodes).get(productSlug);
+    expect(data.title).toEqual('Football');
+  });
+
+  it('should fetch a specified node in specified language', async () => {
+    const lang = 'hr';
+    const { data } = await ss.nodes(nodes).get(productSlug, { lang });
+
+    expect(data.title).toEqual('Nogometna lopta');
+  });
+
+  it('should be of specific shape when a node is fetched', async () => {
+    const expectedProduct = {
+      id: 2856,
+      slug: 'football',
+      // TODO is slugs added recently?
+      slugs: {},
+      sectionId: 59,
+      languageId: 'en',
+      isPublished: true,
+      publishedFrom: '2021-01-21T08:58:00',
+      publishedTo: null,
+      createdAt: '2021-01-21T09:59:15.465864',
+      updatedAt: '2021-01-21T09:59:15.465864',
+      category: { id: 2854, slug: 'sport', name: 'sport', itemCount: null },
+      data: {
+        author: '',
+        title: 'Football',
+        description: '',
+        text: '',
+        whatIsIt: '',
+        modelNumber: '',
+        recommended: false,
+      },
+      seoMetadata: '[]',
+      image: null,
+      images: [],
+      contents: null,
+      connectionStats: null,
+      connections: {},
+      event: null,
+      collectionItems: null,
+      subsections: null,
+    };
+
+    const expectedProductKeys = Object.keys(expectedProduct);
+
+    const res = await ss.nodes(nodes).get(productSlug);
+    const resKeys = Object.keys(res);
+
+    expect(resKeys).toEqual(expectedProductKeys);
+  });
+
+  it('should fetch a node with English translations when English language specified', async () => {
+    const expectedResponse = {
+      author: 'Karlo M',
+      title: 'Football',
+      description: 'A beautiful ball',
+      text: '<p>Go out and play football. </p>',
+      whatIsIt: 'sports item',
+      modelNumber: '3',
+      recommended: false,
+    };
+    // NOT SURE WHICH IS THE DEFAULT LANGUAGE, SO SPECIFYING ENGLISH EXPLICITLY
+    const lang = 'en';
+
+    const { data } = await ss.nodes(nodes).get(productSlug, { lang });
+
+    expect(data).toEqual(expectedResponse);
+  });
+
+  it('should fetch a node with Croatian translations when Croatian language specified', async () => {
+    const expectedResponse = {
+      author: 'Karlo M',
+      title: 'Nogometna lopta',
+      description: 'Prekrasna lopta',
+      text: '<p>Igrajte nogomet</p>',
+      whatIsIt: 'sportski rekvizit',
+      modelNumber: '3',
+      recommended: false,
+    };
+    const lang = 'hr';
+
+    const { data } = await ss.nodes(nodes).get(productSlug, { lang });
+
+    expect(data).toEqual(expectedResponse);
+  });
+
+  it('should fetch a node with Italian translations when Italian language specified', async () => {
+    const expectedResponse = {
```

Comment:
the idea here is to not have Italian translation in SS and have it fallback to default language (english)